model/patch_lifecycle.go

```diff
@@ -289,13 +296,13 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 	if err != nil {
 		return nil, errors.Wrap(err, "Error fetching project opts for patch")
 	}
-	intermediateProject, _, err := LoadProjectInto(ctx, []byte(p.PatchedParserProject), opts, p.Project, project)
+	intermediateProject, err := LoadProjectInto(ctx, []byte(p.PatchedParserProject), opts, p.Project, project)
 	if err != nil {
 		return nil, errors.Wrapf(err,
 			"Error marshaling patched parser project from repository revision “%v”",
 			p.Githash)
 	}
-	config, err := CreateProjectConfig([]byte(p.PatchedProjectConfig))
+	config, err := CreateProjectConfig([]byte(p.PatchedProjectConfig), p.Project)
```

Comment:
should this check version control?

Reply:
If version control is disabled PatchedProjectConfig will always be nil, so checking that won't be necessary. I can however wrap this in an if statement if it is an empty string or not.